pandas/tests/io/test_excel.py

```diff
@@ -1907,10 +1780,10 @@ def test_invalid_columns(self):
             with pytest.raises(KeyError):
                 write_frame.to_excel(path, 'test1', columns=['C', 'D'])
 
-    def test_comment_arg(self):
```

Comment:
There were a few one-off functions like this that didn't previously have the `skip_if_no_xlrd` call in them However, when trying to run them without `xlrd` but say `xlwt` installed they would throw:

`ModuleNotFoundError: No module named 'xlrd'`

thrown by the ExcelFile constructor. So I'm assuming it was an oversight not to have these skipped, (they will be going forward with the class structure I put in place)